src/credential-status/status-token.ts

```diff
@@ -0,0 +1,38 @@
+import { StatusArray } from "./status-array";
+import { StatusList } from "./status-list";
+import { cborEncode } from "../cbor";
+import { CoseKey } from "../cose";
+import { CWT } from "../cwt";
+
+interface CWTStatusTokenOptions {
+    statusArray: StatusArray;
+    aggregationUri?: string;
+    type: 'sign1' | 'mac0';
+    key: CoseKey;
+}
+
+enum CWTProtectedHeaders {
+    TYPE = 16
+}
+enum CWTClaims {
+    STATUS_LIST_URI = 2,
+    ISSUED_AT = 6,
+    STATUS_LIST = 65533
+}
```

Comment:
Can you stay consistent with the other enums we already have, e.g. take a look at  the casing of `CoseKeyParameter`. We always try to stay as consistent as possible within a repository.